src/components/Tabs/Tab/Tab.jsx

```diff
@@ -1,14 +1,31 @@
+import cx from "classnames";
 /* eslint-disable jsx-a11y/anchor-is-valid,jsx-a11y/click-events-have-key-events */
-import React, { useRef, forwardRef } from "react";
+import React, { forwardRef, useRef } from "react";
 import NOOP from "lodash/noop";
 import PropTypes from "prop-types";
-import cx from "classnames";
 import useMergeRefs from "../../../hooks/useMergeRefs";
 import Icon from "../../Icon/Icon";
-import "./Tab.scss";
+import { ELEMENT_TYPES, getTestId } from "../../../utils/test-utils";
+import styles from "./Tab.module.scss";
 
 const Tab = forwardRef(
-  ({ className, id, value, disabled, active, focus, onClick, icon, iconType, iconSide, children }, ref) => {
+  (
+    {
+      className,
+      id,
+      value,
+      disabled,
+      active,
+      focus,
+      onClick,
+      icon,
+      iconType,
+      iconSide,
+      children,
+      "data-testid": dataTestId
```

Comment:
the api will be data-testid or dataTestId?

Reply:
"data-testid" everywhere as the same attribute can by used on html tags